### PR TITLE
adjusted font size and alignment

### PR DESCRIPTION
**Pull request by bipin-shrestha**
# Pull Request Template

Go from a screen that selects the game play of single player or multiplayer.
When the single player mode is selected it goes into the screen which gives the option to read, write, and/or draw. 1. You will be able to check or uncheck draw or write but cannot choose both write and draw together 2. After chooseing you will be able to upload the drawing or writing to the platform.
When it is submited a function will pit the player against a randomly chosen boss.

## Description

Add a summary that address the following:

- motivation/purpose for the feature
  To make a gamemode menu for single player where they can choose read write or draw and then upload their writing or drawing
- What were those changes
  A gamemode menu for uploading a drawing by choosing the button.
- What issues were fixed
  A gamemode styled menu
- Provide the link to the user story it is referencing in Trello board or other supporting document
  https://trello.com/c/Q4Wusyn7/53-create-button-to-select-single-player-mode

## Commits Checklist

- [ ] commits have clear title names
- [ ] commit has descriptive message of what has changed
- [ ] more small commits than less big commits

## Code Checklist

- [ ] code is formatted appropriately
- [ ] code meets DRY standards
- [ ] no dead code (check for logs and print statements)
- [ ] no TODOS instead necessary comments
- [ ] code follows Story Squad standards for:
  - Language
  - Framework
  - Libraries

### PRs needs to be reviewed by at least 2 team members

- TPL is the 3rd reviewer
- Lots of comments and suggestions, please!

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

**Review by milklot — DISMISSED** [review]
Good job, keep the same styling of font will structure more pleasant view.

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.env.sample
```
Comment (tinomen) [inline .gitignore]: Remove this line. The sample is meant to be in the repo for future teammates to get an idea of what env vars exist.

**Review by tinomen — CHANGES_REQUESTED** [review]
Just a quick fix on that `.env.sample` file

**Review by PVigar88 — APPROVED** [review]
Looks good

File: src/components/pages/WordCountContainer/WordCountContainer.styles.css [hunk 2]
```diff
@@ -1,22 +1,28 @@
 .word-count-container {
   margin: 0 auto;
-  width: 400px;
+  width: 525px;
```
Comment (Julianisrel) [inline src/components/pages/WordCountContainer/WordCountContainer.styles.css:3]: Good style changes, nice and clean.

**Review by Julianisrel — APPROVED** [review]

**Comment by derekjpeters** [comment]
@bipin-shrestha is this still valid or does this need to be removed?

**Comment by bipin-shrestha** [comment]
I removed all old pull requests.. cuz I saw others have changed same codes
in thier pull requests.

**Pull request closed without merging**